src/plugin/weekOfYear/index.js

```diff
@@ -2,7 +2,12 @@ import { MS, Y, D, W } from '../../constant'
 
 export default (o, c, d) => {
   const proto = c.prototype
-  proto.week = function () {
+  proto.week = function (week = 0) {
+    if (week > 0) {
```

Comment:
Instead of checking if `week` is positive, you should check if it's undefined.

Momentjs allows set a negative week

``` javascript
console.log(moment('2018-01-20').week(-1).format()); // "2017-12-23T00:00:00+01:00"
```

So I think that we should as well